CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
```

Comment:
[Banned C function detected (strncpy)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS111237.md#DS111237) 7 times
strncpy is dangerous, as if the source contains 'n' or more characters, it will not null terminate the destination.

*Fix Guidance:* Use strcpy_s or strlcpy if possible. If no safe function is viable, strcpy/strncpy should be proceeded by conditional checks to verify tha that the source string will fit in the destination with a null termnator.


Suggestions:
[1] 
[2]